tst/EnergyPlus/unit/AirTerminalSingleDuctMixer.unit.cc

```diff
@@ -7767,4 +7768,415 @@ TEST_F(EnergyPlusFixture, AirTerminalSingleDuctMixer_SimFCU_ATMInletSideTest)
+    zoneEqSizing.ATMixerCoolPriDryBulb = 30.0;
+    finalZoneSizing.ZoneHumRatAtCoolPeak = 0.0075;
+    zoneEqSizing.ATMixerCoolPriHumRat = 0.0095;
+    finalZoneSizing.DesCoolLoad = 10000.0;
+    finalZoneSizing.DesHeatLoad = 10000.0;
+    finalZoneSizing.CoolDesTemp = 12.8;
+    finalZoneSizing.CoolDesHumRat = 0.0085;
+    finalZoneSizing.HeatDesTemp = 40.0;
+    finalZoneSizing.HeatDesHumRat = 0.0075;
+
+    // heating mode tests
+    FanCoilUnits::CoolingLoad = false;
+    FanCoilUnits::HeatingLoad = true;
+    ZoneSysEnergyDemand.allocate(1);
+    auto &zoneSysEnergyDemand(ZoneSysEnergyDemand(1));
+    auto &zoneEquipConfig(ZoneEquipConfig(1));
+
+    // set zone air node conditions
+    Node(zoneEquipConfig.ZoneNode).Temp = 20.0;
+    Node(zoneEquipConfig.ZoneNode).HumRat = 0.0075;
+    Node(zoneEquipConfig.ZoneNode).Enthalpy = Psychrometrics::PsyHFnTdbW(Node(zoneEquipConfig.ZoneNode).Temp, Node(zoneEquipConfig.ZoneNode).HumRat);
+    // primary air conditions
+    DataEnvironment::OutDryBulbTemp = 5.0;
+    DataEnvironment::OutHumRat = 0.005;
+    DataEnvironment::OutEnthalpy = Psychrometrics::PsyHFnTdbW(DataEnvironment::OutDryBulbTemp, DataEnvironment::OutHumRat);
+    // primary air condition set at outdoor air condition
+    Node(thisFanCoil.ATMixerPriNode).Temp = DataEnvironment::OutDryBulbTemp;
+    Node(thisFanCoil.ATMixerPriNode).HumRat = DataEnvironment::OutHumRat;
+    Node(thisFanCoil.ATMixerPriNode).Enthalpy = DataEnvironment::OutEnthalpy;
+    // initialize air terminal mixer primary air mass flow rate
+    PrimaryAirMassFlowRate = 0.1;
+    Node(thisFanCoil.ATMixerPriNode).MassFlowRate = PrimaryAirMassFlowRate;
+    Node(thisFanCoil.ATMixerPriNode).MassFlowRateMaxAvail = PrimaryAirMassFlowRate;
+    // set secondary air (recirculating air) conditions to zone air node
+    Node(thisATMixer.SecInNode).Temp = Node(zoneEquipConfig.ZoneNode).Temp;
+    Node(thisATMixer.SecInNode).HumRat = Node(zoneEquipConfig.ZoneNode).HumRat;
+    Node(thisATMixer.SecInNode).Enthalpy = Node(zoneEquipConfig.ZoneNode).Enthalpy;
+
+    BeginEnvrnFlag = true;
+    ZoneEqFanCoil = true;
+    // check availability manager Night Cycle parameters
+    EXPECT_EQ(SystemAvailabilityManager::ThermostatWithMinimumRunTime, SystemAvailabilityManager::NCycSysAvailMgrData(1).CycRunTimeCntrlType);
+    EXPECT_EQ(DataHVACGlobals::NoAction, SystemAvailabilityManager::NCycSysAvailMgrData(1).AvailStatus);
+
+    // set predicted heating load
+    zoneSysEnergyDemand.RemainingOutputReqToCoolSP = 4000.0;
+    zoneSysEnergyDemand.RemainingOutputReqToHeatSP = 4000.0;
+    zoneSysEnergyDemand.RemainingOutputRequired = 4000.0;
+    QZnReq = zoneSysEnergyDemand.RemainingOutputRequired;
+    QUnitOut = 0.0;
+    QLatOut = 0.0;
+    InitFanCoilUnits(state, FanCoilNum, ZoneNum, ZoneNum);
+    Sim4PipeFanCoil(state, FanCoilNum, ZoneNum, ZoneNum, FirstHVACIteration, QUnitOut, QLatOut);
+    // check results when the fna coil unit is not available
```

Comment:
fna -> fan

Reply:
Let the CI machine run complete and I will do formatting and cleanup.